src/clang-c-frontend/clang_c_language.cpp

```diff
@@ -359,6 +361,7 @@ _Bool __ESBMC_overflow_smulll(long long int, long long int, long long int *);
 _Bool __ESBMC_overflow_umul(unsigned int, unsigned int, unsigned int *);
 _Bool __ESBMC_overflow_umull(unsigned long int, unsigned long int, unsigned long int *);
 _Bool __ESBMC_overflow_umulll(unsigned long long int, unsigned long long int, unsigned long long int *);
+int __builtin_esbmc_sync_fetch_and_add(int*, int);
```

Comment:
I honestly don't like changing the function names using the preprocessor (in some cases it breaks compilation, see the comment below). I still couldn't find a better solution though. 

Please change the name to comply with the others, starting with __ESBMC_